data/vnc.service

```diff
@@ -1,15 +1,8 @@
 [Unit]
 Description=VNC Per-Connection Server
-After=syslog.target
 
 [Service]
-Group=privileged
-Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/activeuser/dbus/user_bus_socket
-Environment=XDG_RUNTIME_DIR=/run/activeuser
 # Include common nemo project wide env settings
 EnvironmentFile=-/var/lib/environment/nemo/*.conf
```

Comment:
Needed in user session?

Reply:
Yeah, it's not. Good thinking. :+1:

Reply:
Actually, as @okodron pointed out, this is needed to read `LIPSTICK2VNC_OPTS` from `/var/lib/environment/nemo/70-lipstick2vnc.conf`.